packages/terra-cell-grid/src/_CellGridUtils.js

```diff
@@ -0,0 +1,37 @@
+const staticStyle = width => (
+  {
+    msFlex: '0 0 0px',
+    flex: '0 0 0px',
+    maxWidth: width,
+    minWidth: width,
+  }
+);
+
+const scalarStyle = width => (
+  {
+    msFlex: `${width} ${width} 0px`,
+    flex: `${width} ${width} 0px`,
+  }
+);
+
+const styleFromWidth = (width) => {
+  if (!width) {
+    return null;
+  }
+  if (width.static) {
+    return staticStyle(`${width.static.value}${width.static.unit}`);
```

Comment:
Should we validate static.value and static.unit is provided? Should we fallback to `px` if unit is not provided?

Reply:
Updating to be required sub-props of static.